business_requirement_deliverable/views/business_view.xml

```diff
@@ -31,14 +31,14 @@
             <field name="model">business.requirement.deliverable</field>
             <field name="arch" type="xml">
                 <tree string="Business Requirement Deliverable" default_order="sequence">
-                    <field name="currency_id" invisible="1"/>
+                    <field name="currency_id" invisible="1" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
```

Comment:
@pedrobaeza There are 3 level of information with different level of ACL in the BR:
* Functional: visible by the BR user
* Sales: visible by estimation group
* Cost: visible by the cost group